instrumentation/spring/spring-boot-resources/library/src/main/java/io/opentelemetry/instrumentation/spring/resources/SpringBootServiceNameDetector.java

```diff
@@ -293,6 +293,11 @@ InputStream openClasspathResource(String filename) {
       return classLoader.getResourceAsStream(path);
     }
 
+    InputStream openClasspathResource(String filename, String location) {
+      String path = location + "/" + filename;
+      return classLoader.getResourceAsStream(path);
+    }
```

Comment:
Now that the `SystemHelper` is used in two classes, WDYT about making it a top-level class?